test/test_ConfigFileParser.cpp

```diff
@@ -366,10 +383,26 @@ TEST_F(ValidConfigFileTests, DirectiveAndClosingBracketOnSameLine)
 
 TEST_F(ValidConfigFileTests, DirectiveAndClosingBracketOnSameLineUnderServerDirective)
 {
-	GTEST_SKIP() << "Skipping because not implemented";
 	ConfigFile configFile;
 	EXPECT_NO_THROW(configFile
 		= m_configFileParser.parseConfigFile("config_files/directive_close_open_bracket_same_line_under_server.conf"));
 	EXPECT_EQ("127.0.0.1", configFile.servers[0].host);
 	EXPECT_EQ("8080", configFile.servers[0].port);
 }
+
+TEST_F(ValidConfigFileTests, LocationPath)
+{
+	ConfigFile configFile;
+	EXPECT_NO_THROW(configFile = m_configFileParser.parseConfigFile("config_files/location_path.conf"));
+	EXPECT_EQ(3, configFile.servers[0].locations.size());
+	EXPECT_EQ("/", configFile.servers[0].locations[0].path);
+	EXPECT_EQ("/hello", configFile.servers[0].locations[1].path);
+	EXPECT_EQ("/world", configFile.servers[0].locations[2].path);
+}
+
+TEST_F(ValidConfigFileTests, DirectiveInheritance)
+{
+	ConfigFile configFile;
+	EXPECT_NO_THROW(configFile = m_configFileParser.parseConfigFile("config_files/directive_inheritance.conf"));
+	EXPECT_EQ("/var/www/html", configFile.servers[0].locations[0].root);
+}
```

Comment:
The PR is missing both config files. Therefore the tests are segfaulting:

![image](https://github.com/user-attachments/assets/b7532dad-7471-4438-aefc-8ef688c3271b)